lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -263,6 +263,9 @@ keys_lookup[113] = "mouse_wheel_down"
 
 registerCallback("destruct",function(self)
 	KeyAlert[self.entity] = nil
+    --Used futher below. Didn't want to create more then one of these per file
+    if SpawnAlert[self.entity] then SpawnAlert[self.entity] = nil; end
+    if LeaveAlert[self.entity] then LeaveAlert[self.entity] = nil; end
```

Comment:
These tables aren't defined at this point, nor are they named the same as the ones defined below